raiden-ts/docs-source/installing-dapp.md

```diff
@@ -24,3 +24,24 @@ yarn workspace raiden-dapp run serve
 Once the development server has been started, navigate to `http://localhost:8080` to start using the Raiden dApp.
 
 > Note that for the Raiden dApp to work it requires MetaMask or some other Web3 provider (wallet apps with dApp support) to be installed on your browser.
+
+## Recover Backup locally
+
+You might have a backup by a dApp instance that was provided by a service that
+is not available anymore. You can not import this backup to another dApp
+provider if you was using a Raiden subkey (configuration option). The subkey
```

Comment:
Were

Reply:
Oh damn, yes of course! Wrote too much Python privately. :see_no_evil:

Reply:
:heavy_check_mark: